scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -110,7 +113,7 @@ task CollectCounts {
 
     runtime {
         docker: "${gatk_docker}"
```

Comment:
As per @LeeTL1220's template, we should be putting in values for `cpu` even in the single threaded case.

Reply:
I didn't check to make sure all of the tasks adhered to the template just yet, since I'm assuming that it might still change.  (Actually, I don't think any of the somatic CNV tasks specify cpu.)  I'll focus on the mem changes in this PR and overhaul the rest later (perhaps once tasks are automatically generated), if you don't mind!

Reply:
@samuelklee Fine by me.